.travis.yml

```diff
@@ -19,8 +19,10 @@ matrix:
       node_js: '8'
       env: WEBPACK_VERSION="2.6.0" JOB_PART=coverage
 before_install:
+  - npm i -g npm@latest
```

Comment:
I've limited the install range to `semver: Patch`. If it's a choice between occasionally breaking a patch version and having to pin vs. constantly having to update versions in 58 repositories, i'll take the former any day of the week personally.